### PR TITLE
Add upgrade instructions

### PR DESCRIPTION
**Pull request by Floppy**

### CODE REVIEW

File: UPGRADING.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+If you have deployed using [deploy.bothan.io](https://deploy.bothan.io) or hosted your own instance on Heroku, you will need to make the following manual changes.
+
+* Log in to your Heroku dashboard, and visit the settings page for your Bothan app.
+* Press the "upgrade stack" button to use the new `heroku-16` stack instead of `cedar-14`.
+* In the buildpacks section, add the following entries:
+  * https://github.com/100yrs/heroku-buildpack-cmake.git
+  * https://github.com/heroku/heroku-buildpack-google-chrome.git
+* Remove the `heroku/ruby` buildpack and re-add it so it is at the BOTTOM of the list.
+* To apply the updates immediately, go to the Deploy tab and in the manual deploy section, deploy the master branch. You will not need to do this again, future deployments will be automated in the usual way.
```
Comment (quadrophobiac) [inline UPGRADING.md]: So this step doesn't appear to be universal. For a configured instance the step you detail applies
screen shot 2017-08-16 at 14 32 52
screen shot 2017-08-16 at 14 32 56
However for an instance I set up using bothan-deploy and then didn't make any additional changes this is the default option
screen shot 2017-08-16 at 14 33 49
So I just connected the account to github and enabled a manual deploy, works as the steps detail.
Should connecting to github for automatic deploys be included in the documentation?

Comment (Floppy) [inline UPGRADING.md]: Just changed docs to reflect that. As for setting up auto-deploys, it's a pain. You can only auto-deploy things you own to Heroku, which means that most people get a one-shot deployment and that's it. It's annoying. Something for a separate issues I think.

File: UPGRADING.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  * https://github.com/100yrs/heroku-buildpack-cmake.git
+  * https://github.com/heroku/heroku-buildpack-google-chrome.git
+* Remove the `heroku/ruby` buildpack and re-add it so it is at the BOTTOM of the list.
+
+#### Redeploying
+
+If your app is automatically deploying from a GitHub repository, to apply the updates immediately:
+  
+* go to the Deploy tab 
+* in the manual deploy section deploy the master branch
```
Comment (quadrophobiac) [inline UPGRADING.md]: So this step doesn't appear to be universal. For a configured instance the step you detail applies
screen shot 2017-08-16 at 14 32 52
screen shot 2017-08-16 at 14 32 56
However for an instance I set up using bothan-deploy and then didn't make any additional changes this is the default option
screen shot 2017-08-16 at 14 33 49
So I just connected the account to github and enabled a manual deploy, works as the steps detail.
Should connecting to github for automatic deploys be included in the documentation?

File: UPGRADING.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+* your app should now be deployed successfully.
+
+If you are not deploying automatically, follow your normal manual deployment process, and the next deployment will use the new settings and should succeed.
```
Comment (quadrophobiac) [inline UPGRADING.md]: AH OK - so is there a way to get the same effect without connecting to github, or is connecting to github/theodi/bothan prerequisite?

Comment (Floppy) [inline UPGRADING.md]: It seems to be a prerequisite. I've not worked this out yet, though I'm sure there must be a way. I want to make it happens for other apps too, so it's on my list.

Comment (quadrophobiac) [inline UPGRADING.md]: Maybe it's not so bad that the github prerequisite becomes part of the documentation though? I'm wondering if these steps are something that could be built into bothan-deploy?

Comment (Floppy) [inline UPGRADING.md]: So these steps won't be needed for any future deployments, they're encoded into `app.json`. However, they need to be done manually for old deployments.

Ideally bothan-deploy would handle it, but I'm not sure how.

Comment (quadrophobiac) [inline UPGRADING.md]: SO I think that this patch might not work for anyone not affiliated with the ODI Github account, looking at the way the Heroku interface phrases it
will-work-with-gh
going to test and see if thats the case

Comment (Floppy) [inline UPGRADING.md]: Thanks, difficult to tell what it's like from the outside. I wonder who we could ask to check it over...

Comment (quadrophobiac) [inline UPGRADING.md]: testing blocked due to https://github.com/theodi/bothan-deploy/issues/10

**Comment by quadrophobiac** [comment]
all tests have passed on this @Floppy - can I merge it in despite the dependency-ci issue? (pretty sure I have seen that recur on other PRs)

**Comment by Floppy** [comment]
Yep, dependency-ci is more advisory than a blocker.

**Pull request merged**